client_plugin/drivers/vfile/vfile_driver.go

```diff
@@ -551,14 +550,29 @@ func (d *VolumeDriver) MountVolume(name string, fstype string, id string, isRead
 		return mountpoint, err
 	}
 
-	// Increase GRef
-	log.Infof("Before AtomicIncr")
-	err = d.kvStore.AtomicIncr(kvstore.VolPrefixGRef + name)
+	log.Debugf("MountVolume: before get lock for global refcount of %s", name)
+	// Get the lock for changing the global refcount
+	lock, err := d.kvStore.CreateLock(kvstore.VolPrefixGRef + name)
+	if err != nil {
+		log.Errorf("Failed to create lock for mounting volume %s", kvstore.VolPrefixGRef+name)
+		return "", err
```

Comment:
I think we should call "lock.ClearLock()" before return error.

Reply:
Same comment as above.